web/src/info_overlay.js

```diff
@@ -20,105 +20,217 @@ import * as util from "./util";
 // set_up_toggler is called.
 export let toggler;
 
+function format_usage_html(...keys) {
+    const get_formatted_keys = () => keys.map((key) => `<kbd>${key}</kbd>`).join("+");
+    return $t_html(
+        {
+            defaultMessage: "(or <key-html></key-html>)",
+        },
+        {
+            "key-html": get_formatted_keys,
+        },
+    );
+}
+
 const markdown_help_rows = [
     {
-        markdown: "**bold**",
-        usage_html: "(or <kbd>Ctrl</kbd>+<kbd>B</kbd>)",
+        markdown: $t(
+            {
+                defaultMessage: "{markdown_syntax}bold{markdown_syntax}",
```

Comment:
Yeah, that's not the simplest way to mark this string for translation. The idea behind this solution is to find a balance by providing context to the translators while also making sure they understand not to edit the markdown syntax.